openassessment/xblock/test/test_studio.py

```diff
@@ -352,3 +357,33 @@ def test_editor_context_assigns_labels(self, xblock):
                 self.assertEqual(criterion['label'], criterion['name'])
                 for option in criterion['options']:
                     self.assertEqual(option['label'], option['name'])
+
+    @scenario('data/basic_scenario.xml')
+    def test_render_studio_with_teamset_names(self, xblock):
+        self._mock_teamset_names_only(xblock)
+        frag = self.runtime.render(xblock, 'studio_view')
+        self.assertTrue(frag.body_html().find('teamset_name_a'))
+
+    @scenario('data/basic_scenario.xml')
+    def test_get_teamset_names(self, xblock):
+        xblock.xmodule_runtime = Mock(
+            course_id='test_course',
+            anonymous_student_id='test_student',
+        )
+        xblock.runtime = Mock()
+        mock_teams_config = Mock(
+            teamsets=[Mock(name="tset1"), Mock(name="tset2")]
+        )
+        mock_team_configuration_service = Mock()
+        mock_team_configuration_service.get_teams_configuration.return_value = mock_teams_config
+        xblock.runtime.service.return_value = mock_team_configuration_service
+
+        team_names = xblock.get_teamset_names("cid")
```

Comment:
Nit: I know it doesn't matter, but maybe change "cid" to "test_course" so it's a little more obvious that you're intending to call this method with the same course_id as the test XBlock's runtime.